### PR TITLE
Explicit build and deploy without using any triggers in BC & DC

### PR DESCRIPTION
**Pull request by Kyubinhan**

### CODE REVIEW

File: openshift/templates/build.bc.yaml [hunk 1]
```diff
@@ -58,44 +67,4 @@ objects:
...
-      dockerStrategy:
-        from:
-          kind: ImageStreamTag
-          name: python:3.6
-          namespace: openshift
-      type: Docker
-    triggers:
-        - type: ConfigChange
-        - type: ImageChange
+    triggers: []
```
Comment (DerekRoberts) [inline openshift/templates/build.bc.yaml:70]: There's no need to keep the line around without any inputs.  Please remove 'triggers: [ ]'.

File: openshift/templates/deploy.dc.yaml [hunk 2]
```diff
@@ -48,22 +48,23 @@ objects:
...
-        matchLabels:
-          app: ${NAME}-${SUFFIX}
+        name: ${NAME}-${SUFFIX}
+      strategy:
+        type: Rolling
+      triggers: []
       template:
         metadata:
           labels:
-            app: ${NAME}-${SUFFIX}
```
Comment (DerekRoberts) [inline openshift/templates/deploy.dc.yaml:65]: Was this removed on purpose?  app=${NAME}-${SUFFIX} is used for cleanup.

File: openshift/templates/build.bc.yaml (hunk 1 above)
Comment (Kyubinhan) [reply openshift/templates/build.bc.yaml:70]: This is how you disable all the triggers. If we remove this, there is gonna be a config change trigger as a default

File: openshift/templates/deploy.dc.yaml (hunk 2 above)
Comment (Kyubinhan) [reply openshift/templates/deploy.dc.yaml:65]: I think I removed this because it doesn't work with DeploymentConfig.

**Comment by Kyubinhan** [comment]
Closing the pr, these changes are already merged through #35

**Pull request closed without merging**